sqsworkers/crew.py

```diff
@@ -153,7 +161,10 @@ def run(self):
             'connected to sqs, approx. %s msgs on queue' %
             self.queue.attributes['ApproximateNumberOfMessages']
         )
-        self.poll_queue()
```

Comment:
this belongs in the public `poll_queue` method, not here

Reply:
https://github.com/goodmanship/sqsworkers/pull/11#discussion_r274059564

Reply:
Got it. Making this change.